lib/ShareManifests.js

```diff
@@ -0,0 +1,108 @@
+    "iconURL": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAMAAAAoLQ9TAAAAbFBMVEX////gSj+qODDgSj/o5tiwr6ThVEniZ1zn08XOzMCqODDmtajl49bY18nkjoLdST7Lyr2/VUuvoJblopXWRzzFxLjT0cW6aV/mv7LjhXjPRDq0fXPCQDfMS0Hg3tHe3M60sqfl49W7ua6vkYfYzKRYAAAAA3RSTlMA19csuNBxAAAAaElEQVR4Xp3IRw7CMBQFwJj3m51e6f3+dyTYoGSJmOVkP3Bq9GHq5gAkT5ELDjGA6h0VcAox9oCaKeptSEE9IILhyN+gnQDnJy9Bpm3H6yj9yOvwJdG0RNFcaHa9pXBoCoruj7DJ/vACm58E7AMvk10AAAAASUVORK5CYII=",
+    "name": "Gmail",
+    "origin": "https://mail.google.com",
+    "pageSize": {
+      "share": {
+        "height": 500,
+        "width": 500
+      }
+    },
+    "shareURL": "https://mail.google.com/mail/?view=cm&ui=2&tf=0&fs=1&body=%{body}&su=%{title}",
+    "postActivationURL": "https://activations.cdn.mozilla.net/en-US/activated/gmail.html"
+  },
```

Comment:
Same comment as above re: required `version` key:

```
social.manifest.mail-google.com:
  child "version" fails because ["version" is required]
```